style.css

```diff
@@ -454,8 +454,10 @@ textarea:focus {
 
 .common-li {
   color: #fff5e1;
-  font-size: 15px;
+  font-size: 33px;
+  font-weight: bold;
   padding-bottom: 16px;
+  border-bottom: solid 1px #6f6c6b;
```

Comment:
[OPTIONAL] Kindly try to match options in the desktop version to match the Figma design 
| Your design | Figma design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/157025374-1ef03554-bdf5-45e9-a7b2-98491ffd05a7.png) | ![image](https://user-images.githubusercontent.com/24703171/157024514-4d820c72-25dc-4fcd-8851-5a912f9e66c1.png) |